croaring.lua

```diff
@@ -0,0 +1,38 @@
+project "croaring"
+
+dofile(_BUILD_DIR .. "/static_library.lua")
+
+configuration { "*" }
+
+uuid "C8804B2B-1A46-4E89-955A-92AEFB6CDBB3"
+
+defines {
+
+}
+
+includedirs {
+  "include",
+  "cpp"
+}
+
+files {
+  "src/*.c",
```

Comment:
Do you know if every file needs to be built? We usually just build the bare minimum needed for consumption to reduce the amount of fixes we need for all the platforms we support

Reply:
So I talked with Jason from the kernel team and he is telling me that they use them all.